app/src/main/java/com/x8bit/bitwarden/ui/vault/feature/item/VaultItemLoginContent.kt

```diff
@@ -158,6 +158,7 @@ fun VaultItemLoginContent(
                 Spacer(modifier = Modifier.height(8.dp))
                 NotesField(
                     notes = notes,
+                    onCopyAction = vaultCommonItemTypeHandlers.onCopyNotesClick,
                     modifier = Modifier
                         .testTag("CipherNotesLabel")
```

Comment:
Shouldn't this `testTag` be passed in like you did with the `IdentityCopyField`?